be/src/common/config.h

```diff
@@ -456,7 +456,9 @@ CONF_mInt64(write_buffer_size, "104857600");
 // user should set these configs properly if necessary.
 CONF_Int64(load_process_max_memory_limit_bytes, "107374182400"); // 100GB
 CONF_Int32(load_process_max_memory_limit_percent, "30");         // 30%
-CONF_Int64(compaction_mem_limit, "2147483648");                  // 2G
+CONF_Int64(compaction_max_memory_limit_bytes, "-1");
```

Comment:
compaction_max_memory_limit

Reply:
Fixed

Reply:
I think compaction_max_memory_limit_bytes is better. 
It is consistent with compaction_max_memory_limit_percent below.